tools/fv_surf_map.F90

```diff
@@ -523,7 +523,8 @@ subroutine FV3_zs_filter (bd, isd, ied, jsd, jed, npx, npy, npx_global,  &
       real(kind=R_GRID), intent(IN):: stretch_fac
       logical, intent(IN) :: bounded_domain
       real, intent(inout):: phis(isd:ied,jsd,jed)
-      real, intent(inout):: oro(isd:ied,jsd,jed)
+      !real, intent(inout):: oro(isd:ied,jsd,jed)
+      real, intent(in):: oro(isd:ied,jsd,jed)
```

Comment:
Why did this need to be changed?

Reply:
The moving nest code calls FV3_zs_filter() after a nest move, and its call tree has that source orography data as intent(in).  Since the FV3_zs_filter() subroutine does not seem to modify the values of oro, I changed the call to the stricter intent(in).  The internals of FV3_zs_filter() were not changed.